ui/src/main/webapp/src/app/analysis-context/analysis-context-form.component.html

```diff
@@ -89,8 +89,10 @@ <h2 *ngIf="!project" i18n>Loading...</h2>
                     <ng-container i18n>Identifying packages...</ng-container>
                 </p>
             </div>
-            <wu-js-tree-wrapper [treeNodes]="packageTree" [(selectedNodes)]="analysisContext.includePackages">
-            </wu-js-tree-wrapper>
+            <div class="fixed-height">
+                <wu-js-tree-wrapper [treeNodes]="packageTree" [(selectedNodes)]="analysisContext.includePackages">
+                </wu-js-tree-wrapper>
+            </div>
```

Comment:
That is also another possibility. But I guess I prefer the way you solved it now, because we might want to set some `max-width` and maybe some other properties and that component could potentially get huge amount of inputs and would have lot of conditions.